.pre-commit-config.yaml

```diff
@@ -16,8 +16,8 @@ repos:
     rev: v1.32.0
     hooks:
       - id: golangci-lint
-  - repo: https://github.com/prettier/prettier
-    rev: 2.1.2
+  - repo: https://github.com/prettier/pre-commit
```

Comment:
Thanks for your information, I change to use it and seems fine.
```
➜  modelmesh-serving git:(fix-prettier) ✗ pre-commit run --all-files prettier
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-prettier:prettier@2.1.2.
[INFO] Installing environment for https://github.com/pre-commit/mirrors-prettier.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
prettier.................................................................Passed
```